title.tex

```diff
@@ -1,4 +1,4 @@
-\title{Practical Volume Rendering using Programmable Shaders in VTK for Scientific and Medical Computing}
+\title{High Performance Ubiquitous Volume Rendering using Programmable Shaders in VTK for Scientific and Medical Computing}
```

Comment:
@aashish24 I know I said leaving high performance but on second thought I'd say 

"Ubiquitous volume rendering using programmable shaders in VTK for scientific and medical [visualization]" has a nice ring to it - thoughts?

Minor - no need to capitalize intermediate words in the title.